src/discussions/threads/thread/Thread.jsx

```diff
@@ -0,0 +1,99 @@
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { faQuestionCircle, faStar as faEmptyStar } from '@fortawesome/free-regular-svg-icons';
+import {
+  faCircle, faComments, faFlag, faStar as faSolidStar, faThumbtack,
+} from '@fortawesome/free-solid-svg-icons';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Routes } from '../../../data/constants';
+import messages from './messages';
+
+function Thread({ thread, intl }) {
```

Comment:
This component doesn't match the mockups.
- There's no star at the bottom of the card
- The [A] should be the user's avatar (or a placeholder with dummy image)
- The black dot should be the online/offline status (looks off right now)
- There's no indication that a post is pinned (pin in the icon on mockups)

**Mockup:**
![image](https://user-images.githubusercontent.com/27893385/91783989-140bc380-ebd8-11ea-80b4-f24b0bd8aa57.png)

**Implemented:**
![image](https://user-images.githubusercontent.com/27893385/91783999-1b32d180-ebd8-11ea-8302-52aacc6266c3.png)

Reply:
The star at the bottom of the card is something that was introduced by Kshitij in #1. @xitij2000, should the star be removed or is that something we want in the Thread UI?

I've replaced the `[A]` with a round image that currently sources a random flower image.

The black dot is labelled in the code (from the previous PR) as the read/unread status. @xitij2000 could you provide context as to whether the dot is intended to be read/unread status, or online/offline status for the user. If it is supposed to be read/unread status, under which circumstances should the circle be visible, and what color should it be in those circumstances?

There is in fact code for the pinned status of a post. It used to show up on the right side, but will now be displayed at the bottom alongside the star and comment count.

Reply:
@pcockwell As I mentioned earlier I implemented the original skeletal code based on the older mockups. The basic sturcure is roughly the same, but if in doubt follow the mockups I shared, not my implementaiton.

Reply:
> The black dot is labelled in the code (from the previous PR) as the read/unread status. @xitij2000 could you provide context as to whether the dot is intended to be read/unread status, or online/offline status for the user. If it is supposed to be read/unread status, under which circumstances should the circle be visible, and what color should it be in those circumstances?
It is the unread status. It shows up if unread, and doesn't show up if read. There is no other colour.